include/picongpu/particles/shapes/PCS.hpp

```diff
@@ -75,16 +75,16 @@ struct PCS : public shared_PCS::PCS
              */
             float_X const abs_x = algorithms::math::abs( x );
 
-            bool const below_1 = abs_x < 1.0_X;
-            bool const below_2 = abs_x < 2.0_X;
+            bool const isWithin_1_0 = abs_x < 1.0_X;
+            bool const isWithin_2_0 = abs_x < 2.0_X;
```

Comment:
Is there a reason these names differ from the 1st and 2nd radii in the P4S shape?